src/tls_openssl.c

```diff
@@ -201,6 +201,13 @@ int tls_stop(tls_t *tls)
     int error;
     int ret;
 
+    /* tls/openssl: don't call SSL_shutdown() after a fatal error
```

Comment:
I meant that this message should be in the commit message. That one you write when you run `git commit`. The previous comment here was fine.

The reason I suggested to re-write commit message is because many tools work better with a short 1st line. For example, github will cut a long line. Besides, `git log` is very powerful command when you investigate a problem with code and good commit messages save time.

Also, I would like you to update your 1st commit instead of creating 2nd one. It not only would make git log more tidy, but also allows to use commands `git log|blame|show` efficiently.

So, how you can easily put all changes into 1 commit:
```
# Note that you reset commits till the commit with this hash
git reset c40b4c12c1f81b7c7b4b673cbf19a4ecb4f0c6ab
git add src/tls_openssl.c
git commit
git push -f origin check-for-fatal-error-before-SSL_shutdown
```

If you needed to update only the last commit, you would need to use `git commit --amend` command. This is very useful command if you want to create tidy git logs :).

In conclusion, write the previous comment here, it was fine, make a single commit instead of 2 and move current message to the commit message. If you have any questions, please ask.